requirements.txt

```diff
@@ -2,3 +2,4 @@ colorama==0.3.7
 docopt==0.6.2
 feedparser==5.2.1
 requests==2.20.0
+tqdm=4.48.2
```

Comment:
`tqdm` needs adding to the `setup.py` too.

Reply:
Thanks, I merged the latest version from brtmr:master including this change, it should be good now!